docs/shared/plugin-features/use-code-generators.md

```diff
@@ -1,18 +1,16 @@
-# Using Generators
+# Use Code Generators
 
 ## Overview
```

Comment:
I would just remove the title `## Overview` here and have the paragraph as the intro to the doc